src/preprocessor.ts

```diff
@@ -28,6 +29,9 @@ export function process(src, path, config) {
                 fileName: path
             });
 
+        // strip root part from path
+        path = path.replace(root, '');
```

Comment:
- Add a short note in the comment about why this is needed and leave a link to #158 
- using something like 
```
path = path.startsWith(root) ? path.substr(root.length) : path;
```
might be better